events/community-meeting.md

```diff
@@ -0,0 +1,73 @@
+
+All meetings are archived on the [Youtube Channel](https://www.youtube.com/watch?v=onlFHICYB4Q&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ)
+
+Quick links:
+
+- [Agenda document](https://k8s.devstats.cncf.io/)
+- [Zoom meeting link](https://zoom.us/my/kubernetescommunity)
+
+## Purpose
+
+The purpose of the community meeting is the dissemination of information throughout the project. 
+Large changes in an area of Kubernetes that affect groups of people should be mentioned here. 
+
+
+## Meeting Agenda
+
+If you have a topic you'd like to present or would like to see discussed,
+please propose a specific date on the [Kubernetes Community Meeting Agenda](https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit#).
+
+General speaking the meeting is structured like this:
+
+- Introduction by Host (~3 minutes)
+- Community Demo (~10 minutes)
+- Release Updates
+  - Development Release
+  - Stable Release and point releases
+  - Older stable releases and point releases
+- Graph of the Week (~2 minutes)
+  - This is meant to bring attention to different [devstats graphs](https://k8s.devstats.cncf.io/)
+- SIG Updates
+  - Three SIGs per meeting, 10 minutes per SIG 
+- Announcements (~5 minutes)
+  - Any other community announcements should go here
+
+## Demos
+
+The first 10 minutes of a meeting is dedicated to demonstrations from the community. 
+These demos are noted at the top of the community document. 
+There is a hard stop of the demo at 10 minutes, with up to 5 more minutes for questions.
+Feel free to add your demo request to the bottom of the list, then one of the organizers will get back to you to schedule an exact date. 
+Demo submissions MUST follow the the requirements listed below. 
+
+### Requirements
+
+This meeting is the largest meeting in the community, and is attended by many people, therefore we ask that you _prepare ahead of time_ and not waste this opportunity.
+
+- Your demo MUST NOT be a commercial pitch and MUST be related to Kubernetes. Commercial software and services can be demo'ed but _keep it classy_ and relevant to the audience.
+- You MUST show up to the community meeting _10 minutes_ prior to the start to test your audio and screensharing with the hosts. If you are not present on time, you will _lose your spot_ and be kicked back to the end of the demo queue. 
+- You MUST also commit to being available the week before your demo date in case there is an issue with that week's demo. 
+- The use of a headset or other high quality gear is _strongly_ recommended, typing from the same laptop you are speaking into is distracting
+- Ensure you are presenting from a quiet environment.
+- If you run out of time and your demo is a smashing hit we encourage you to schedule follow on events. 
+
+## SIG Updates 
+
+SIGs will give a community update at least once per release cycle per the [schedule](TBD). 
```

Comment:
Even though the schedule is TBD, this line might be here for a while during the determination phase.  I suggest adding now some words around the trajectory of the scheduling, for example:  "The intention going forward is to have a predictable, fixed schedule of which SIGs presenting in the community during which week of the release cycle.  This allows each SIG to know and plan its obligation and also lets the audience know when to get their regular update from any given SIG."